includes/utils/class-amp-image-dimension-extractor.php

```diff
@@ -172,38 +172,7 @@ private static function determine_which_images_to_fetch( &$dimensions, &$urls_to
 	 * @param string $mode Whether image dimensions should be extracted concurrently or synchronously.
 	 */
 	private static function fetch_images( $urls_to_fetch, &$images, $mode ) {
-		// Use FasterImage when for compatible PHP versions
-		if ( 'synchronous' === $mode ||
-			false === function_exists( 'curl_multi_exec' ) ||
```

Comment:
This `curl_multi_exec()` call gives me pause. Is it possible for this function to not exist in 5.4, say if an older version of cURL is used? We've seen something like this when an older version of ICU is present: https://github.com/ampproject/amp-wp/pull/1440

Reply:
`curl_multi_exec()` should be available on any PHP5 version as the PHP docs state, at least their is no distinction between different sets of `curl_*()` functions. In other words, if `curl_exec()` exists, we can also assume `curl_multi_exec()` exists. Note that WordPress core does that as well, in their cURL implementation they use `curl_multi_exec()` too and don't specifically check its existance. I think we should move forward with this.

Reply:
Apparently this turned out to not be true: https://github.com/ampproject/amp-wp/pull/2183#issuecomment-491506514